.github/workflows/checks.yml

```diff
@@ -158,7 +158,7 @@ jobs:
           git diff --exit-code || (echo 'go.mod/go.sum deps changes detected, please run "make gotidy" and commit the changes in this PR.' && exit 1)
       - name: CodeGen
         run: |
-          make -j2 generate
+          make -j2 gogenerate && make -j2 licence-update
```

Comment:
[**For reviewers**] `gogenerate` removes the licences and `license-update` adds them back. Check if license is missing in a PR is done before so license specific bugs are covered there.